`make securedrop-core-5.15` failed at sed step

```
...
  INSTALL debian/linux-libc-dev/usr/include
dpkg-deb: building package 'linux-libc-dev' in '../linux-libc-dev_5.15.159-1-grsec-securedrop-1_amd64.deb'.
dpkg-deb: building package 'linux-image-5.15.159-1-grsec-securedrop' in '../linux-image-5.15.159-1-grsec-securedrop_5.15.159-1-grsec-securedrop-1_amd64.deb'.
dpkg-deb: building package 'linux-image-5.15.159-1-grsec-securedrop-dbg' in '../linux-image-5.15.159-1-grsec-securedrop-dbg_5.15.159-1-grsec-securedrop-1_amd64.deb'.
make[1]: Leaving directory '/kernel/linux-5.15.159'
sed -i s/#DEB_VERSION_UPSTREAM#/5.15.159-1-grsec-securedrop/ debian/linux-image-5.15.159-1-grsec-securedrop linux-headers-5.15.159-1-grsec-securedrop linux-image-5.15.159-1-grsec-securedrop-dbg securedrop-grsec/DEBIAN/postinst
sed: can't read debian/linux-image-5.15.159-1-grsec-securedrop: No such file or directory
sed: can't read linux-headers-5.15.159-1-grsec-securedrop: No such file or directory
sed: can't read linux-image-5.15.159-1-grsec-securedrop-dbg: No such file or directory
sed: can't read securedrop-grsec/DEBIAN/postinst: No such file or directory
make: *** [debian/rules:32: binary-arch] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
```

It's only supposed to edit one file...

https://github.com/freedomofpress/kernel-builder/blob/32dae88a30b9b398cbae35a40ddf12c37248f304/debian/rules#L38

This is used across both 5.15 and 6.6 but my 6.6 build went fine. Also I'm now realizing that METAPACKGE is spelled wrong -.-

--- a/debian/rules
+++ b/debian/rules
@@ -20,8 +20,8 @@ ifneq (,$(filter-out parallel=1,$(filter parallel=%,$(DEB_BUILD_OPTIONS))))
 endif
 
 # Get the metapackage name by listing the packages from d/control and
-# picking the only one that starts with SD
-METAPACKGE := $(shell dh_listpackages | grep "securedrop")
+# picking the only one that starts with "securedrop"
+METAPACKGE := $(shell dh_listpackages | grep "^securedrop")
 
 .PHONY: binary binary-indep binary-arch
 binary: binary-arch binary-indep

